jekyll/_cci2/docker-layer-caching.md

```diff
@@ -17,15 +17,11 @@ Consider reusing the unchanged layers to significantly reduce image build times.
 
 ``` YAML
 - setup_remote_docker:
-    reusable: true    # default - false
+    docker_layer_caching: true
     exclusive: true   # default - true
 ```
 
-## **`reusable`**
-
-When `reusable` is set to `true`, CircleCI will try to reuse your Remote Docker Environment. That is, every layer you built in a previous job will be accessible in the remote environment and CircleCI will attempt to reuse the previous environment when it is possible. However, in some cases your job may run in a clean environment, even if the configuration specifies `reusable: true`.
```

Comment:
we still need to document what `docker_layer_caching` does and what to expect from it.